### DISCUSSION TITLE
APK?

### DISCUSSION THREAD

**Discussion by j5k** [organicmaps/organicmaps#197] [Q&A]
Is there a way to get the apk? Or do I need to build myself? I would be happy to test it.

**Comment by starsep** [comment]
Disclaimer: I am not a member of a development team

There is early access version in Google Play: https://play.google.com/store/apps/details?id=app.omaps

**Comment by rtsisyk** [comment]
You can download APK from GitHub Release page: https://github.com/organicmaps/organicmaps/releases/tag/2021-04-24

**Reply by j5k** [reply]
Thanks a lot!!

**Comment by tangsgod** [comment]
And from f-droid, that's where i got it from :)

**Reply by j5k** [reply]
Yes, since some days it is also available in the F-Droid store.
